tests/integration/helpers.bash

```diff
@@ -257,6 +260,12 @@ function requires() {
 				skip_me=1
 			fi
 			;;
+		cgroups_devices)
+			init_cgroup_paths
+			if [ ! -e "${CGROUP_DEVICES_BASE_PATH}/devices.list" ]; then
```

Comment:
This check should be removed for v2

Reply:
This check is actually not needed (I added it when experimenting with things), thanks for pointing it out!

Reply:
Fixed!